pytest/tests/sanity/upgradable.py

```diff
@@ -164,8 +164,13 @@ def test_upgrade() -> None:
     assert 'error' not in res, res
     assert 'Failure' not in res['result']['status'], res
 
-    hex_account_balance = int(
-        nodes[0].get_account(hex_account_id)['result']['amount'])
+    # We have seen test failures, where account was not found.
+
+    response = nodes[0].get_account(hex_account_id)
+
+    assert 'result' in response, f'got error from node: {response["error"]}'
+
+    hex_account_balance = int(response['result']['amount'])
```

Comment:
we have 16 places in our tests, where we use "['result']['amount']" -- maybe we should have a util function for that ? (this way it could check that account exists etc etc).

Reply:
We have a few options:
1) Add a check to `get_account`

@mm-near @mina86 Another way is to add a check inside
`get_account`.

```
    def get_account(self, acc, finality='optimistic'):
        res = self.json_rpc('query', {
            "request_type": "view_account",
            "account_id": acc,
            "finality": finality
        })
        assert 'error' not in res, res

        return res
```

2) add argument  `assert_error=True` to `self.json_rpc`. 
Though, this may break some existing code if there is an error

3) add argument  `print_error=True` to `self.json_rpc`. 
Though, this add some extra prints to `json_rpc` shouldn't break anything, but will give us proper debug output.

My vote is on 2), but that may be a breaking change.
1) should be safe to do.